### DISCUSSION TITLE
0.5.10

### DISCUSSION THREAD

**Discussion by z-huang** [z-huang/music#1521] [Announcements]
- Multi-select in library songs, artist songs, albums, and history
- Search in history
- Auto scroll up or down when reordering
- Show other versions of an album
- Add cancel button for download notification
- Better audio normalization
- Option to disable screenshot for privacy
- Option to disable auto-loading more songs
- Other fixes and enhancements

<hr /><em>This discussion was created from the release <a href='https://github.com/z-huang/InnerTune/releases/tag/v0.5.10'>0.5.10</a>.</em>

**Comment by aaru111** [comment]
please add option to disable auto generated suggestions and playlists 
also add top charts please

**Reply by Darkrock04** [reply]
Add more songs

**Reply by Tech123sf** [reply]
Great app but there is no notification of the app on notification bar and lock screen. To change song I have to always open app.

**Reply by Teboho79** [reply]
How to update to new version

**Reply by sanishkarkee** [reply]
Can we play offline music with it?

**Reply by Hikari2w2** [reply]
> Can we play offline music with it?

Yes, you can download the songs and play them offline, however, be aware that Innertune is not working right now, as the developer hasn't been active for months. You should use Outertune's fork, it's constantly being updated and has a ton of new features like local songs playback, yt sync and more.

**Reply by Harsh10025** [reply]
Thank you soo much 🙏 thank you 😊

**Reply by kunal987678** [reply]
> please add option to disable auto generated suggestions and playlists also add top charts please

Unknown error coming

**Reply by AkshatRaj00** [reply]
It's not working 😞

**Reply by nicolasordoguate** [reply]
A mí no me funciona

**Comment by birmehto** [comment]
Wow bro dedication 👏

**Comment by FL1kkeR** [comment]
Babe, wake up, another legendary innertune update just dropped!

**Reply by nicolasordoguate** [reply]
A mí no me funciona

**Comment by Anonuser-code** [comment]
https://github.com/user-attachments/assets/00918f1e-d917-45a4-954f-774bffa7a364

**Comment by unseenlarks** [comment]
thank you, @z-huang.

**Comment by Hikari2w2** [comment]
Thank you bro

**Comment by jojosloes** [comment]
My dude going so fast, thanks for keeping this project up man
(Also is auto download in the options or is it not (easily) doable)

**Comment by serrq** [comment]
A release with very good features!

**Comment by Roxo7777** [comment]
A question: does backup backup playlist? I think it's interesting to just make an information bubble saying which things will be stored up in backup

**Reply by z-huang** [reply]
Yes.

**Comment by suvojeet-sengupta** [comment]
> * Multi-select in library songs, artist songs, albums, and history
> * Search in history
> * Auto scroll up or down when reordering
> * Show other versions of an album
> * Add cancel button for download notification
> * Better audio normalization
> * Option to disable screenshot for privacy
> * Option to disable auto-loading more songs
> * Other fixes and enhancements
> 
> _This discussion was created from the release [0.5.10](https://github.com/z-huang/InnerTune/releases/tag/v0.5.10)._

Awesome bro

**Comment by ghost** [comment]
I have a few requests 
1. Please add the option to disable automatic queue, I want to be able to play 1 song that I clicked without next songs coming in automatically 
2. Please fix the most played to be 50 items/songs and it's stability.

Thank You, Love the app Xo!

**Comment by A-Tetarwal** [comment]
Can we have a shuffle option from song history , like the one which we have for library songs

**Comment by Melo-5** [comment]
Amazing update Z! Thanks a lot..

I have some suggestions/features I've experienced with another app that will improve greatly the overall experience, in this case from OuterTune:

1. YouTube Music sync: shown in History tab:

![Screenshot_2024-09-02-12-12-18-609-edit_com dd3boh outertune](https://github.com/user-attachments/assets/8aaf64fb-9543-4ff9-bcab-1ce8a74cfb80)

-

![Screenshot_2024-09-02-12-19-31-801-edit_com dd3boh outertune](https://github.com/user-attachments/assets/4e96cb42-b109-4e34-9dab-ed6e10234f72)

And 2. Player with Blur effect and option to tap album cover once for Lyrics: 

![Screenshot_2024-09-02-12-15-17-540-edit_com dd3boh outertune](https://github.com/user-attachments/assets/66c18c3b-ecd5-4440-bd41-fc3a9c3fa466)

Thanks again for making this one of the few worthy apps out there!!

**Reply by ruskcoder** [reply]
The yt music sync already has a issue. Make an issue request for the blur, it keeps everything organized.

**Comment by Xisrr1** [comment]
Do you plan on making syncing liked songs from YT music possible?

**Comment by Roxo7777** [comment]
In the next version, when you search for some song in the playlist, select all the songs that appear in the search result. 

https://github.com/user-attachments/assets/575b0bfb-4533-4b52-9698-4bd3ba4170c0

Note that the correct number of songs that appear in the search result is 117, however, when I press the button to select all musics, select all of musics from the playlist, not from the search results.

**Reply by Roxo7777** [reply]
This will be very useful for me to select songs from artists I've saved in a playlist.

**Comment by pillownara** [comment]
Thank you as always for the update! Really happy with this app.

**Comment by Bruvd1** [comment]
What a great discovery for me. a literal artwork this is, thank you creator😊

**Comment by Pooc09** [comment]
I ran into a problem when updating the application to version 0.5.10, some songs start to sound strange, as I understand it is due to the audio normalization function, because if I turn it off, the music goes normally without extraneous hum, but unfortunately much quieter, I did not encounter such a problem on earlier versions, I hope in 0.5.11 the problem will be solved.

**Reply by Borwonyc** [reply]
Same Here, Thought It Were My External Equalizer App Weighing The Sound Along The Dobly Atmost Yet No. Otherwise Pretty Neat, No Crashes. All It's Features Are Top Notch With No Additions Please. Might Mess The Code With GUI Improvements Like The Blur Request. Please Keep It This Simple. Probably Only Play With The Track Belt Only.

**Comment by Jackpot7z** [comment]
Thank you so much for the constant support!

**Comment by chavishcb** [comment]
Yay. THANK YOU 😊. Is there a way to have the album art offline [ only for downloaded songs] ? It will be easier to navigate and find songs.

**Comment by sumanth3821** [comment]
New Bug!!!

When I am playing the song in background if I pause the song and keep it idle for 20 to 30 min 

![Screenshot_2024-09-05-06-32-10-637_com miui home](https://github.com/user-attachments/assets/d869e250-6413-475d-bab4-fc72fe5c12a4)
 
The song gets stopped and the play button dont work, i have to go back to app and play it again. 
Please solve this issue.

**Reply by ruskcoder** [reply]
Make an issue for it.

**Reply by Roxo7777** [reply]
Sign out and in from the app. I tested it on my device, it doesn't happen the same.

**Reply by Roxo7777** [reply]
Friend, this is not necessarily a bug, it may be due to RAM memory or background settings;
Try going to InnerTune's battery settings, so look for options that involve not suspending it
Or, if it has "Appear over others"

**Reply by Borwonyc** [reply]
Keep It Unrestricted In Battery.

**Comment by Ghgcbjb** [comment]
there is a problem with the sound of the music

**Reply by iexbse** [reply]
Same

**Reply by chavishcb** [reply]
Yes suddenly the volume of track increases. I have set my music quality to "high". Even tough I face this issur

**Comment by iexbse** [comment]


**Reply by ruskcoder** [reply]
It might be a phone issue.

**Reply by iexbse** [reply]
I dont think so because every app works just fine

**Reply by iexbse** [reply]
I tried to install it on my tablet and my songs somehow were already there and the audio is no better

**Comment by Jameswahid** [comment]
Hlw there I have a request. Could we have an account option in the app? When we delete the app, all the data, especially the playlist, gets deleted as well. if we have an account option, we wouldn't have to suffer from this.Please

**Reply by ruskcoder** [reply]
Backup and restore...?

**Reply by IRBIS-18** [reply]
i once tried the backup and restore function when i was switching phones . it didnt work . then i manually created those playlists again

**Reply by Jameswahid** [reply]
> Backup and restore...?

It doesn't work for the playlists

**Comment by dadone99** [comment]
Please can you add the ability to add a widget to the phone home? I think this can further implement this app

**Comment by Daixi156** [comment]
![Screenshot_20241003-072342.png](https://github.com/user-attachments/assets/2c859cbe-48e2-442f-b2ff-eb355cb344fe)

**Comment by Borwonyc** [comment]
Stable At Best!. Please Do Not Add Anymore Features, Please Keep The Code Neat And Tidy, Focusing On Performance, Stability & Usage.
None! Decorations i.e GUI Background Blurs, Custome Wallpapers, Equalizers, Widgets Etc.

Apart From The Track Belt 😁 Like The New Ones.

**Comment by luizcarlos95** [comment]
friend, there is a bug happening where the songs are not being played and it is more or less like this: I search for a song and the song is not played, it gives an error or I go to see other songs even the ones in my library and they do not play. I think it would be good to release a small update without any additions but if you are going to add it let it be something very small so as not to give an error in the songs

**Reply by Roxo7777** [reply]
Report the bug, 
The update paused, probably it will come back only next year, I still don't know,
But I hope he will catch a lot of the bugs and features.

**Comment by luizcarlos95** [comment]
quando não tiver mais esses bugs eu volto a usar

**Comment by aceydot** [comment]
I've added Serbian in the translations. Hopefully it can be added in the app?

**Comment by varuosb** [comment]
I have an issue with song loading even if my wifi is fast it shows "time out" or "unknown error". I can't figure out why??

**Reply by varuosb** [reply]
InnerTune_v0.5.10_full_arm64-v8a.apk

**Reply by varuosb** [reply]
I tried it. Still same as before

**Reply by varuosb** [reply]
But it works in the 5.9 version. So I switched back. So I think there is some issues with the latest version

**Reply by luizcarlos95** [reply]
> Which apk did you download?

I downloaded the latest version 5.10 right here on Github

**Reply by luizcarlos95** [reply]
> Hmm...works well for me. Can you give more details about the error? Like can you load songs or can you just play them?

simply no song in the entire app does not play at all I downloaded the latest version 5.10 and it gave an error I downloaded 5.9 and it gave an error I downloaded it from fdroid and it gave an error every version I download gives an error maybe it's on my phone but I don't think so because I recently bought it from the Samsung physical store if you have any doubts to know what model my phone is it's the a35 256GB 8RAM

**Reply by Hikari2w2** [reply]
Have you tried using a VPN?, I've had a similar issue and was able to solve it by using VPN

**Comment by Snapxxx12** [comment]
![Screenshot_2024-10-16-22-32-26-350_com.zionhuang.music.jpg](https://github.com/user-attachments/assets/3d954395-194c-4819-b6c9-78fde89de19b)

real

**Reply by luizcarlos95** [reply]
mine is the same thing but in all the songs a few months ago there was this bug in the app in some songs but there were very few but now almost all the songs give an error

**Reply by Shahria7** [reply]
Facing same issue

**Reply by Newworldr** [reply]
I'm also having same problem

**Reply by Newworldr** [reply]
How do I solve this issue 😔 help me out someone

**Reply by Melo-5** [reply]
Just go to settings>Content> Log in.
Use your non-primary google account for safety purposes.
Problem solved.

**Reply by ardwiinoo** [reply]
It's because YouTube Music change it's API, it happen also in other open source platform such as SimpMusic which is already fixed.

**Comment by luizcarlos95** [comment]
z please release a newer version without adding anything but a newer version to fix the bugs that are happening

**Comment by harshitv804** [comment]
Need an auto-download option, where once it's downloaded it should show in the library or download section. I really miss that feature since ViMusic.

**Comment by Shahria7** [comment]
I found the solution for **please sign in** issue.
Just go to settings>Content> Log in. 
Use your non-primary google account for safety purposes. 
Problem solved.

**Comment by Gbrayant** [comment]
![Screenshot_20241021-232402_InnerTune](https://github.com/user-attachments/assets/47a47d67-e592-4f02-b2e7-714b0c44a85f)

**Reply by akshay5687** [reply]
Yes me too facing the same sign in problem

**Comment by WARbitCOIN** [comment]
> - Multi-select in library songs, artist songs, albums, and history
> - Search in history
> - Auto scroll up or down when reordering
> - Show other versions of an album
> - Add cancel button for download notification
> - Better audio normalization
> - Option to disable screenshot for privacy
> - Option to disable auto-loading more songs
> - Other fixes and enhancements
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/z-huang/InnerTune/releases/tag/v0.5.10'>0.5.10</a>.</em>

**Comment by Ruerujei0622** [comment]
Please include for next updates is the indicator if the music is already in a playlist thank you

**Reply by Roxo7777** [reply]
It already exists, when you try to put it in a playlist, there's a stop, showing what you already have, and having the options to put duplicate or ignore (not put), although it would be useful to have some sort of badge to show on already being in one.

**Comment by MountainEgg** [comment]
discord rich presence doesn't seem to work

**Comment by GoodDoctorrrr** [comment]
It's always requesting to login even after i did it . Cannot play some songs due to these problem 😕  faced by me

**Comment by Z2430** [comment]
On Oppo A94 5G I have the problem that when I close the app even in the background the icon at the top always remains active and so I have to go to the app info and close every time

![IMG_20241130_090531.jpg](https://github.com/user-attachments/assets/5b23be1c-0094-480a-8634-32eafc2fc923)

![IMG_20241130_091103.jpg](https://github.com/user-attachments/assets/2cc33277-9519-4148-815d-583620272dd1)

**Comment by Darkrock04** [comment]
U have to clear the played song in the app

**Reply by Z2430** [reply]
I didn't know about this. Solved, many thanks

**Comment by nabakun24** [comment]
Hello admin please can u add Spotify playlist feature thanks loving your app❤

**Comment by En07A1** [comment]
Till yesterday the app was working fine today there's been a glitch all the songs are showing error cant download any or play any. Pls look into this

**Reply by Hikari2w2** [reply]
> Till yesterday the app was working fine today there's been a glitch all the songs are showing error cant download any or play any. Pls look into this

Try with VPN

**Reply by En07A1** [reply]
Still the same issue

**Reply by En07A1** [reply]
Tried reinstalling it using vpn diff google account but still not solved

**Reply by sooncriime** [reply]
I also have the same problem and I don't know how to Fix it.

**Reply by Hikari2w2** [reply]
> Still the same issue 

Actually, i just opened the app and getting the same problem, cant play any song. Im also getting the same error with other forks lile OuterTune, seems YT api made some severe changes.

**Reply by En07A1** [reply]
Yaa well we can only wait till inner or outertune do something

**Reply by Hikari2w2** [reply]
> Yaa well we can only wait till inner or outertune do something 

Yeah, hopefully they'll fix it soon...

**Reply by sooncriime** [reply]
Hopefully

**Reply by JH3RSON** [reply]
![Screenshot_20241217-155332_InnerTune.png](https://github.com/user-attachments/assets/76d2559c-a181-4904-9db8-ef716b6b2cee)
I have the same problem, does anyone know why?

**Reply by Darkrock04** [reply]
The developer needs an maintenance

**Comment by ruskcoder** [comment]
It seems like YouTube might have made API changes. There isn't much to do other than wait, this repo isn't updated as often anymore. Maybe use the fork called OuterTune, they might have a fix soon enough.

**Reply by mikoarizola** [reply]
> It seems like YouTube might have made API changes. There isn't much to do other than wait, this repo isn't updated as often anymore. Maybe use the fork called OuterTune, they might have a fix soon enough.

apparently outertune has that problem too (i downloaded it) do you have any more alternative? since i have a lot of backup songs in that app

**Reply by Rence-ai** [reply]
Bro I've downloaded the universal apk and it works. I can play any music there, but you must sign in first to play music.

**Reply by mikoarizola** [reply]
> Bro I've downloaded the universal apk and it works. I can play any music there, but you must sign in first to play music.

yes i can stream bro, but the problem is some songs are not downloading and some songs stops at 30 seconds , dang i really love this app i hope someone know a alternatives for this, because i think this app  is abandoned i literally have 3000+ songs in this and i can't download 85% of them

**Reply by En07A1** [reply]
Download outertune you can use backup from innertune for it and its working well

**Reply by mikoarizola** [reply]
> Download outertune you can use backup from innertune for it and its working well

download not working there bro, I've tried it some songs are not downloading too

**Reply by En07A1** [reply]
Idk why its not working for you i am using it rn while writing this and its working fine without a single problem

**Reply by En07A1** [reply]
I am using ver 0.7.0 alpha 2

**Reply by IvnoGood** [reply]
If you want to download songs search for python scripts based on yt music i know one that works rlly well

**Comment by JH3RSON** [comment]
![Screenshot_20241217-155332_InnerTune.png](https://github.com/user-attachments/assets/0add55d1-ed02-40d2-9648-a367e85a4c10)

I haven't been able to listen to any song for a few hours, because when I try to play one I get this problem Does anyone know why?

**Reply by dihyannn** [reply]
I have same problem

**Reply by sooncriime** [reply]
Does anyone have a solution?

**Reply by EB1FLy** [reply]
Me too

**Reply by Rence-ai** [reply]
Same situation🙄

**Comment by JUDESONJOBY** [comment]
![Screenshot_20241218_184447](https://github.com/user-attachments/assets/45e8929f-4aef-47af-ba56-a2751e57d4f1)
Having this problem in several phones, any solutions?

**Reply by Zenith0xYT** [reply]
Just download a forked version for now

**Reply by Medivir72exe** [reply]
> Just download a forked version for now

How do you exactly do that

**Reply by Awoo4** [reply]
> Just download a forked version for now

How to do it

**Comment by ghost** [comment]
do it support android 15

**Reply by Xcatalogue** [reply]
yes

**Comment by Bobsmilk** [comment]
what is up with the dif apk files for this app and which ones should i use with my phone?

**Comment by 0KiRa1** [comment]
App not working

**Reply by 0KiRa1** [reply]
Please update it

**Reply by ganairm** [reply]
its probably abandoned
use the forks, the 2 that are the most stable right now are right here:
https://github.com/mostafaalagamy/Metrolist/releases/tag/v10.8.0
https://github.com/Malopieds/InnerTune/releases/tag/v0.5.13

**Reply by mikoarizola** [reply]
> its probably abandoned use the forks, the 2 that are the most stable right now are right here: https://github.com/mostafaalagamy/Metrolist/releases/tag/v10.8.0 https://github.com/Malopieds/InnerTune/releases/tag/v0.5.13

none of them worked sadly it's having a same problem with innertube as of today

**Reply by En07A1** [reply]
Outertune

**Comment by manjit32987** [comment]
Bro app is not working

**Reply by meamdumy** [reply]
Go for metrolist, it works

**Reply by Darkrock04** [reply]
Go for outertune

**Comment by 0KiRa1** [comment]
Please update the app, it is not working.

**Reply by I-luv-silly-cats** [reply]
If anyone has any alternative apps pls tell us🙏🏼🙏🏼

**Reply by beswalod** [reply]
> If anyone has any alternative apps pls tell us🙏🏼🙏🏼

You can use some forks. This for example: https://github.com/Malopieds/InnerTune

**Reply by gmwangi23** [reply]
Having a similar problem cannot play any music what to do?

**Reply by meamdumy** [reply]
> Having a similar problem cannot play any music what to do?

Just use a yt music hacked client. They now allow downloads and are pretty good. It doesn't work at all for me either and I just use that option. Just make sure to use virus total to scan the file for viruses beforehand. *Ahem* Winulator.

**Reply by mikoarizola** [reply]
> > If anyone has any alternative apps pls tell us🙏🏼🙏🏼
> 
> You can use some forks. This for example: https://github.com/Malopieds/InnerTune

not working too sadly some songs says "unknown error" ughh i love this app so much i hope they fixed it? may i ask if u find any more alternative that works i tried outertune but it doesn't work too some songs says unknown error

**Reply by mikoarizola** [reply]
> > Having a similar problem cannot play any music what to do?
> 
> Just use a yt music hacked client. They now allow downloads and are pretty good. It doesn't work at all for me either and I just use that option. Just make sure to use virus total to scan the file for viruses beforehand. _Ahem_ Winulator.

send link please so i can download it while this one is not fixed😭😭

**Comment by redmonkeybutt** [comment]
It's been quite some time I have been trying to play a song but it keeps showing me unknown error. Please help me. @z-huang 
![Screenshot_2024-12-30-11-46-16-10_f4703f2d42345ac28c56f11a6100c500](https://github.com/user-attachments/assets/189ed344-d46c-4d8b-aca8-195b46a4ec03)

**Reply by meamdumy** [reply]
Same here
![Uploading Screenshot_20241230-211333.InnerTune.png…]()

**Comment by Rence-ai** [comment]
Useless app, it doesn't playing a music, it's just appearing 'Unknown Error' even I logged in to this app. Bro, here's your problem.![Screenshot_20250101-215304.png](https://github.com/user-attachments/assets/19031584-0a46-4794-8bda-97bc07251bbd)

**Comment by Titan40-ui** [comment]
Like everybody is facing unknown errors so please release a new version . Cause we are not able to play, add or download new musics even after signing in . 
Hope the developers will fix it soon , 😌 Till then we will have to work with outertune tune only . 
Thanks!

**Comment by mikoarizola** [comment]
This problem has been 3 weeks problem now the problem is when i download my 1600+ songs i can't download some of them 20% of the songs, and after the download some songs that are not able to download either it says "error unknown" or "no stream available" please i really love this app so much update it please😭 i cant download offline songs no more

**Comment by Kryptonbattery** [comment]
Why does it show an unknown error when I click to play music?

**Reply by Kryptonbattery** [reply]
I can only play the music I downloaded

**Reply by mikoarizola** [reply]
> I can only play the music I downloaded

same, don't delete the app because once you delete it you can nevee be able to download the song again since it will say " Download failed" i literally deleted my 3000 song because of 1 song is saying "error unknown" or "no stream available" now when i tried downloading all of them 85% of my library songs are not downloading 😭

**Comment by KenGato** [comment]
Send link to version 0.70 alpha

**Comment by Jameswahid** [comment]
Admin you should do something.please fix the unknown errors issue.its already been 4 months and this app still not getting updated

**Reply by Hikari2w2** [reply]
> Admin you should do something.please fix the unknown errors issue.its already been 4 months and this app still not getting updated 

admin already dead 💀🙏

**Comment by Rence-ai** [comment]
Better use the OuterTune I think it's better than InnerTune.
This app has no problem at all. Than k u developer for making this app.
![Screenshot_20250118-055125](https://github.com/user-attachments/assets/9a173c8c-e2ed-4253-9655-816ac7b3dc06)

**Comment by Wiz6rd** [comment]
Hello i have a problem... When i logged in i cant play any song and it just says "Unknown error"

**Comment by tghjklo** [comment]
I can't listen to music on InnerTune anymore:

![Uploading IMG20250312121732_copy_1013x1351.jpg…]()

There's always a "source error" when wanting to hear music on Android Auto. I want to hear songs from **YouTube community playlists**.

Can someone please help?

Btw: My phone is on Android 14 and I'm using InnerTine version 0.5.10 (full). I had the same problem with the fork version

**Reply by meamdumy** [reply]
It's broken, and this isn't new. Try a forked app like OuterTune. Things like these are still updated regularly and are pretty good.

**Reply by tghjklo** [reply]
Thanks for the quick reply, I already tried a few other apps like OuterTune, RiMusic, Harmony, SimpMusic (all those also work with Android Auto), but none of them was working with YouTube playlists.

**Reply by Hikari2w2** [reply]
> Thanks for the quick reply, I already tried a few other apps like OuterTune, RiMusic, Harmony, SimpMusic (all those also work with Android Auto), but none of them was working with YouTube playlists.

Have you already tried Outertune's latest update?

**Reply by rcky844** [reply]
You may try [my builds](https://github.com/z-huang/InnerTune/discussions/1741#discussioncomment-12469218)

**Comment by Vasanth-kv** [comment]
The Latest update 0.5.10 version is having issues like I couldn't able to play the music if clicked for playing it says "UNKNOWN ERROR" and the play button is appears but no music and if I download any music it goes on loading but couldn't download single music. So fix it soon so everyone can enjoy the music. 

![IMG_20250704_095907](https://github.com/user-attachments/assets/5b26c984-5eba-41ca-ad3d-b3e2a5097097)
![IMG_20250704_095809](https://github.com/user-attachments/assets/24c44713-6692-4248-9433-694e6b86e6c2)

**Comment by Vasanth-kv** [comment]
Thank you for your email
How can use the innertune app is the app will get update
And you guide me to the steps you mentioned in mail

**Comment by mike9245** [comment]
> * Multi-select in library songs, artist songs, albums, and history
> 
>     * Search in history
> 
>     * Auto scroll up or down when reordering
> 
>     * Show other versions of an album
> 
>     * Add cancel button for download notification
> 
>     * Better audio normalization
> 
>     * Option to disable screenshot for privacy
> 
>     * Option to disable auto-loading more songs
> 
>     * Other fixes and enhancements
> 
> 
> _This discussion was created from the release [0.5.10](https://github.com/z-huang/InnerTune/releases/tag/v0.5.10)._

 I can't play any songs. Getting the unknown error other than that I can see the songs while searching. Can anyone tell me how to fix this?

**Comment by khanamit27632-lab** [comment]
I am not able to play song

**Comment by khanamit27632-lab** [comment]
Unknown error showing any solution

**Comment by vitorroman17** [comment]
Hello, I am from Brazil and all the songs are making mistakes, is there a problem? may not work in my country?

**Comment by AkshatRaj00** [comment]
> * Multi-select in library songs, artist songs, albums, and history
> * Search in history
> * Auto scroll up or down when reordering
> * Show other versions of an album
> * Add cancel button for download notification
> * Better audio normalization
> * Option to disable screenshot for privacy
> * Option to disable auto-loading more songs
> * Other fixes and enhancements
> 
> _This discussion was created from the release [0.5.10](https://github.com/z-huang/InnerTune/releases/tag/v0.5.10)._

Hey guys, I built OneMusic because I was done with annoying ads and subscription pop-ups while listening to music.It’s a lightweight, open-source music app designed purely for the community.What you get:No Subscriptions Needed 🚫Absolutely Add-Free 🛑No Signup/Login required – Privacy first 🔒If you want a clean, no-nonsense music experience, give it a shot:🌐 Website: https://onemusic-website.vercel.app/
💬 Get latest updates on Discord: [https://discord.com/invite/sfXpprH5]

**Comment by polupanovavv-tech** [comment]
Добрый день, напомните пожалуйста что у вас не работает?

чт, 27 авг. 2026 г., 09:01 nicolasordoguate ***@***.***>:
